webapp/src/actions/index.js

```diff
@@ -230,3 +233,25 @@ export function handleInstanceStatusChange(store) {
         });
     };
 }
+
+export function sendEphemeralPost(store, message, channelId) {
```

Comment:
nit: do we have to write this? nothing in the mattermost webapp and/or redux that we can (should?) use? Also, does this come from the right user?

Reply:
@levb That's a good question. There is a sendEphemeralPost in webapp's actions/global_actions.jsx. I tried briefly but couldn't get it to work. The zoom plugin had to repeat this code, so I followed its lead.
@crspeller Do you know of a way to use the webapp function?

Reply:
The way to reuse that at the moment would be to move the function to mattermost-redux then import it.

Reply:
eww... I don't want to do that...  :)

Reply:
Maybe it's a good ticket, but probably not for this one.

Reply:
@levb re: right userId -- I'm not sure that it matters for ephemeral posts. Looks like the webapp function doesn't even set a userId: https://github.com/mattermost/mattermost-webapp/blob/9488f4fa2b92500e7c1254c58443b94a38ceda47/actions/global_actions.jsx#L181

The zoom plugin uses the `state.entities.users.currentUserId` like I do here.